interfaces/cython/cantera/utils.pyx

```diff
@@ -23,7 +24,8 @@ cdef pystr(string x):
 
 def add_directory(directory):
     """ Add a directory to search for Cantera data files. """
-    CxxAddDirectory(stringify(directory))
+    directory = Path(directory)
+    CxxAddDirectory(stringify(str(directory)))
```

Comment:
What's the purpose in casting to a `Path` and then immediately back to a string?

Reply:
That way, you can add a `Path` in `cantera.add_directory` ... see `__init__.py`.

Reply:
But if it's passed in as a `Path`, you don't need to convert it to a `Path`, and `str` on a string is a no-op, right?

Reply:
`stringify` on a `Path` raises an error, hence the need to convert `Path` to `str`

Reply:
No, I get that. The question is about why `directory = Path(directory)`.

Reply:
I guess a simple `directory = str(directory)` would suffice. Ugh!